### ISSUE TITLE
RegEx DoS attack in debug@2.x.x package

### ISSUE THREAD

**Issue by MrBenJ** [get-alex/alex#229]
### Subject of the issue

There's a low severity vulnerability issue with the `debug` package used in babel 6 and Ava for tests.

### Your environment

*   MacOS 10.13.5
*   Node v8.2.1 npm v6.1.0

### Steps to reproduce

Start a new project with `npm init`
Install `alex` with `npm install alex`
Notice the `npm audit` security vulnerability appear.

### Expected behaviour

There should be no security vulnerabilities when installing `alex`

### Actual behaviour

There is a low severity security vulnerability.

### Offer of assistance

I can refactor the tests and upgrade `ava` to `ava@next` which uses Babel 7, and remove the old dependency of debug v2 to remove this vulnerability.

If this offer of assistance sounds good, go ahead and assign this issue to me and I'll take care of it right away!

Thank you for creating a wonderful project to contribute to!

[remark]: https://github.com/remarkjs/remark

[retext]: https://github.com/retextjs/retext

[equality]: https://github.com/retextjs/retext-equality

[profanities]: https://github.com/retextjs/retext-profanities

**Comment by wooorm** [comment]
I think this issue isn’t really a problem, as this is just stuff user for development. It’s not like we’re running a server that could crash (in which case this would be a problem).

I do think that other people will see this warning and take it too literally though, so I’m fine with upgrading to ava@next 👍

**Comment by MrBenJ** [comment]
I totally agree with you on that @wooorm - The reason I'm bringing this up is that Github and npm both show big warnings on my repos that use your project. They look a little like this: 
![screen shot 2018-10-11 at 9 03 19 am](https://user-images.githubusercontent.com/7918387/46817771-8c431480-cd34-11e8-8613-a570f738dc47.png)

While it's alarming, it really isn't that bad of an issue, since like you said, Alex isn't running a server or similar.

I'll go ahead and fix this up for you. PR incoming before end of day today :) (USA time)

**Comment by MrBenJ** [comment]
Hello!

I found the root cause of the `npm audit` issue. It's here:
https://github.com/alessioalex/git-spawned-stream/pull/5

I submitted a PR to that project as well. Those tests are passing right now and hopefully we can get this all taken care of :D.

As a quick side note, there's this great free tool I use to keep dependencies up to date called [Greenkeeper](https://greenkeeper.io/). 

Thanks again! Appreciate you maintaining this awesome project :)

**Comment by alessioalex** [comment]
Issue fixed in `git-spawned-stream`, published version `1.0.1` to npm already.

**Comment by wooorm** [comment]
Perfect! OK if I close this?

**Comment by alessioalex** [comment]
fine by me, sure!
